package.json

```diff
@@ -20,8 +20,10 @@
     "@hennge/vue3-pagination": "^1.0.17",
     "@heroicons/vue": "^1.0.6",
     "axios": "^0.27.2",
+    "crypto-js": "^4.1.1",
     "date-fns": "^2.28.0",
     "moment": "^2.29.3",
+    "npm": "^8.8.0",
```

Comment:
@akmiller01 I'm thinking this was an accident?

Reply:
Not intentional, but it's how npm decided to modify my package-lock.json when I installed crypto-js

Reply:
Reversing it here: https://github.com/IATI/datastore-search/pull/207